Script/char.dat

```diff
@@ -445,9 +445,9 @@ farmer
     "\"The prices are infamous here. Ivan should smack that capitalist shopkeeper hard!\"";
   }
   AttachedGod = SEGES;
-  ScienceTalkPossibility = 33;
+  ScienceTalkPossibility = 20;
   ScienceTalkIntelligenceModifier = 5;
-  ScienceTalkWisdomModifier = 10;
+  ScienceTalkWisdomModifier = 2;
```

Comment:
Right now farmers increase your Int a bit more, while housewives increase your Wis a bit more. That's why farmers' Wis modifier was decreased so much.

Should I still increase it? Should I increase the Int modifier of housewives to 5 as well?